src/Symfony/Component/Workflow/Validator/WorkflowValidator.php

```diff
@@ -12,13 +12,38 @@
 namespace Symfony\Component\Workflow\Validator;
 
 use Symfony\Component\Workflow\Definition;
+use Symfony\Component\Workflow\Exception\InvalidDefinitionException;
 
 /**
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
 class WorkflowValidator implements DefinitionValidatorInterface
 {
+    private $singlePlace;
+
+    /**
+     * @param bool $singlePlace
+     */
+    public function __construct($singlePlace = false)
+    {
+        $this->singlePlace = $singlePlace;
+    }
+
     public function validate(Definition $definition, $name)
     {
+        if ($this->singlePlace) {
+            foreach ($definition->getTransitions() as $transition) {
+                if (1 < count($transition->getTos())) {
```

Comment:
This was, as is, accepted before.. and is actually yoda style. Not sure we should change it now..